scripts/build.sh

```diff
@@ -6,6 +6,8 @@ set -e
 DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 cd ${DIR}/..
 
+target=${1:-*}
```

Comment:
omg. i never thought of this. instead for years i've been begrudgingly rebuilding everything even when i needed only one binary.